enterprise-suite/es-monitor-api/bootstrap-monitors.json

```diff
@@ -571,6 +571,69 @@
         "description": "Memcached evictions in {{$labels.workload}} is not normal on {{$labels.instance}}",
         "monitorType": ""
       }
+    },
+    {
+      "model": "sma",
+      "parameters": {
+        "name": "akka_http_server_response_time",
+        "metric": "akka_http_http_server_response_time_ns{quantile=\"0.5\"}",
+        "period": "15m",
+        "numsigma": "2",
+        "minval": "100000000",
+        "window": "15m",
+        "confidence": "0.5",
+        "severity": "warning",
+        "summary": "HTTP server response time is anomalous",
+        "description": "{{$labels.app}} on {{$labels.instance}} has unusual response time",
+        "monitorType": ""
+      }
+    },
+    {
+      "model": "sma",
+      "parameters": {
+        "name": "akka_http_client_response_time",
+        "metric": "akka_http_http_client_http_client_service_response_time_ns{quantile=\"0.5\"}",
+        "period": "15m",
+        "numsigma": "2",
```

Comment:
How are these values chosen?

Reply:
following akka_processing_time

is a very conservative:  numsigma 2 means 5% error, 15 minutes is our slow window (5, 10, 15), 100ms is our ignore line, ignoring anomalies under 100ms

Reply:
Is it documented anywhere how the values were attained?

Just thinking about how we would explain these things to customers. And also how we add new monitors with sensible values. Why 5% standard error? Is median a good indicator of poor health on response times (which are usually long tailed)?